Nextcloud/Extensions/DependencyInjection.cs

```diff
@@ -14,11 +14,21 @@ public static IServiceCollection AddNextcloud(this IServiceCollection services,
         byte[] authToken = Encoding.UTF8.GetBytes($"{username}:{password}");
         var auth = new AuthenticationHeaderValue("Basic", Convert.ToBase64String(authToken));
 
+
         services.AddHttpClient<NextcloudCircleClient>(i =>
```

Comment:
At least in my c# version I have to include the Interface when registering the service. Otherwise I get the error: Unable to resolve service for type 'Nextcloud.Interfaces.INextcloudCollectivesClient